### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by Homa3030**

### CODE REVIEW

**Comment by Maxkoz777** [comment]
Well done!

**Comment by none-word** [comment]
Good job!

**Comment by afonyaa** [comment]
LGTM!

File: monitoring/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      options:
+        max-size: "100m"
+        max-file: "10"
+        tag: "{{.ImageName}}|{{.Name}}"
+    networks:
+      - monitoring.network
+    logging:
+      driver: "json-file"
+      options:
+        tag: "{{.ImageName}}|{{.Name}}"
```
Comment (AKurmazov) [inline monitoring/docker-compose.yml:21]: nit: This statement just overrides the same one above, you can safely remove it

**Comment by AKurmazov** [comment]
Good job! 👍

**Pull request closed without merging**